### ISSUE TITLE
export interface from a module out put warning logs in to the console.

### ISSUE THREAD

**Issue by jason-chang** [typescript-eslint/typescript-eslint#670]
**What code were you trying to parse?**

```ts
// Please put code here

// selection.ts
export interface SelectionState {
  before: string,
}

// index.ts
export { Selection, SelectionState } from './selection';
```

**What did you expect to happen?**
No error and no warning will out put in to the console.

**What actually happened?**

	 warning  in ./src/index.ts

	"export 'SelectionState' was not found in './selection'
	
**Versions**

| package                     | version |
| --------------------------- | ------- |
| `@typescript-eslint/parser` | `1.7.0` |
| `TypeScript`                | `3.4.5` |
| `ESLint`                    | `5.16.0` |
| `node`                      | `10.16.0` |
| `npm`                       | `6.9.0` |

**Comment by bradzacher** [comment]
What is the rule that's unexpectedly erroring for you?
The error message should have a rule name with it.

**Comment by jason-chang** [comment]
I found that it's a bug with webpack or ts-loader.
